### ISSUE TITLE
update from 4.3.1

### ISSUE THREAD

**Issue by justin-ven** [GoogleChrome/workbox#2374]
Using workbox-cli to generate precache.  Updating to workbox v5.0.0 from v4.3.1 (npm install workbox-cli --global) and entering "workbox injectManifest workbox-config.js" at command line 
produced error requesting "self.__WB_MANIFEST" be included in serviceworker.  Editting service worker by replacing workbox.precaching.precacheAndRoute([]); with workbox.precaching.precacheAndRoute(self.__WB_MANIFEST); resolved this error, and inspection of the serviceworker file suggests that the injection worked.  However, testing updated site in Chrome produced console error, stating: 

Uncaught TypeError: t is not iterable
    at u.addToCacheList (PrecacheController.js:55)
    at U (precache.js:50)
    at Object.precacheAndRoute (precacheAndRoute.js:26)
    at serviceworker:76 (this is the line where the precache has been injected)

Serviceworker registration failed.

Reverting to v4.3.1 corrected this issue.

**Comment by philipwalton** [comment]
Can you share the generated service worker file (and source map if you have it)? If so, that would help a lot in reproducing the issue.

**Comment by justin-ven** [comment]
Hi Philip,

Find attached the pre-processed service worker file, with v5.0.0 lines commented out (due to the problems we encountered).  I would need to re-create the error to obtain the post-processed file (as we did not save it), but a quick check suggested that the pre-cache did work and the post-processed file was similar to post-process.js attached.

Best,

Justin.

**Comment by philipwalton** [comment]
It looks like you tried to attached the file in an email, but that doesn't work with GitHub comments. Can you upload it somewhere publicly? E.g. glitch.com or a similar service?

**Comment by justin-ven** [comment]
Hi again Philip,

Apologies for the delay – I didn’t have time to look at this yesterday (my time).  Recreating the error, I found a problem with our implementation, which caused the console error that we found for the service worker.  The issue concerning the injection point remains, but that is a minor problem (fixed by replacing [] with self.__WB_MANIFEST in workbox.precaching.precacheAndRoute([])).

Regards,

Justin.
